yarn.lock

```diff
@@ -5455,27 +5029,11 @@ foreground-child@^1.5.6:
     cross-spawn "^4"
     signal-exit "^3.0.0"
 
-form-data@^3.0.0:
```

Comment:
@ilmotta : iOS builds seem to be failing because this dependency was removed from the project in this PR.
cc @yakimant @jakubgs : perhaps we should explicitly include it in our package.json since we need it for DIAWI uploads?

Reply:
should be fixed here : https://github.com/status-im/status-mobile/pull/18299